src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -296,6 +296,8 @@ public abstract boolean recompileBuildFile(File buildFile, List<String> artifact
     private String dockerRunOpts;
     private volatile Process dockerRunProcess;
     private File defaultDockerfile;
+    private List<String> srcMount;
+    private List<String> destMount;
```

Comment:
Since src and dest are two things that always go together I suggest these arrays be replaced by one HashMap. This would avoid the values getting out of synch if something unforeseen caused one of the values in one of the arrays to be dropped or not inserted.

Reply:
Originally, I was actually thinking of using a HashMap. But the issue with using a HashMap is technically a `src` could be repeated, because a user could potentially copy the same file to multiple locations. That would lead to duplicate keys, which you cannot have in a regular <String, String> HashMap. So I looked into how to handle multiple keys in a HashMap, which led to things I thought were maybe too excessive for this particular use case.

We can discuss further if you would like. @ericglau  any thought on this?

Reply:
HashMap would not work for all cases here, such as the below.
```
COPY a b
COPY a c
```
A better data structure could be used though to prevent them getting out of sync, such as a 2d array or an array of objects (where each object has src and dest).  But I think it's ok as is.